applications/asset_tracker_v2/src/main.c

```diff
@@ -388,7 +388,8 @@ static void data_get(void)
 		app_module_event->data_list[count++] = APP_DATA_MODEM_STATIC;
 	}
 
-	if (!app_cfg.no_data.neighbor_cell || !app_cfg.no_data.gnss || !app_cfg.no_data.wifi) {
+	if (IS_ENABLED(CONFIG_LOCATION_MODULE) &&
+	    (!app_cfg.no_data.neighbor_cell || !app_cfg.no_data.gnss || !app_cfg.no_data.wifi)) {
```

Comment:
The idea has been to set the `CONFIG_DATA_SAMPLE_*` Kconfig options to `n` so this has worked.
Except there's been a bug with WiFi as there wasn't a configuration for that so that should be fixed anyway.

Reply:
So we'll keep IS_ENABLED(CONFIG_LOCATION_MODULE) here because you could easily enable some method from cloud and the build wouldn't support location stuff. Also, this is easier to read from the code.